test/test.js

```diff
@@ -853,6 +855,67 @@ describe('node-fetch', () => {
 		});
 	});
 
+	it('should allow POST request with URLSearchParams as body', function() {
```

Comment:
You can only enable these tests on Node.js v7.0.0+. To do so you could put

```js
const itUSP = typeof URLSearchParams === 'function' ? it : it.skip;
```

and use `itUSP` instead of `it` here.